test/storage/online_file_source.test.cpp

```diff
@@ -149,7 +149,7 @@ TEST(OnlineFileSource, TEST_REQUIRES_SERVER(RetryDelayOnExpiredTile)) {
     std::unique_ptr<AsyncRequest> req = fs.request(resource, [&](Response res) {
         counter++;
         EXPECT_EQ(nullptr, res.error);
-        EXPECT_GT(util::now(), res.expires);
+        EXPECT_GT(util::now(), *res.expires);
```

Comment:
Otherwise we catch this compilation error using the Android toolchain:
```
../../../mason_packages/android-x86-9/gtest/1.7.0/include/gtest/gtest.h:1585:28: error: invalid operands to binary expression ('const std::__ndk1::chrono::time_point<std::__ndk1::chrono::system_clock, std::__ndk1::chrono::duration<long long, std::__ndk1::ratio<1, 1> > >' and 'const std::experimental::__library_fundamentals_v1::optional<std::__ndk1::chrono::time_point<std::__ndk1::chrono::system_clock, std::__ndk1::chrono::duration<long long, std::__ndk1::ratio<1, 1> > > >')
GTEST_IMPL_CMP_HELPER_(GT, >);
~~~~~~~~~~~~~~~~~~~~~~~~~~~^~
../../../mason_packages/android-x86-9/gtest/1.7.0/include/gtest/gtest.h:1562:12: note: expanded from macro 'GTEST_IMPL_CMP_HELPER_'
  if (val1 op val2) {\
      ~~~~ ^  ~~~~
```
It seems the compiler from Android toolchain is unable to properly handle `std::experimental::optional` dereferencing.